Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -305,6 +305,12 @@
           "Name": "DoNotExposePlaintextSecrets/ConvertToSecureString",
           "ContentsRegex": "$SEC101/052.ConvertToSecureString",
           "MessageArguments": { "secretKind": "ConvertTo-SecureString powershell" }
+        },
+        {
+          "Id": "SEC101/054",
```

Comment:
Because there's another validator waiting in a PR with that number

---
In reply to: [591877697](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591877697) [](ancestors = 591877697)